### PR TITLE
backup & restore

### PR DESCRIPTION
**Pull request by Beiancaa**

### CODE REVIEW

File: lib/tasks/backup_restore.rake [hunk 1]
```diff
@@ -0,0 +1,84 @@
+namespace :db do
+ DUMP_FMT = 'p' # 'c', 'p', 't', 'd'
+
+ desc 'Dumps the database to backups'
+ task dump: :environment do
+   dump_sfx = suffix_for_format(DUMP_FMT)
+   backup_dir = "/home/beianca/db_backups"
```
Comment (MwMaki) [inline lib/tasks/backup_restore.rake]: @Beiancaa please correct the buckup folder name and remove you name from the path

File: lib/tasks/backup_restore.rake [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+ desc 'Restores the database from a backup using PATTERN'
+ task :restore, [:pat] => :environment do |task,args| 
+  if Rails.env.production?
+    puts "cannot restore db"    
+  else
+    if args.pat.present?
+      cmd = nil
+      with_config do |app, host, db, user|
+        backup_dir = "/home/beianca/db_backups"
```
Comment (MwMaki) [inline lib/tasks/backup_restore.rake]: @Beiancaa please correct the buckup folder name and remove you name from the path

File: lib/tasks/backup_restore.rake (hunk 1 above)
Comment (ssharba) [reply lib/tasks/backup_restore.rake]: @Beiancaa backup folder MUST be inside the application "tmp" folder

File: lib/tasks/backup_restore.rake (hunk 2 above)
Comment (ssharba) [reply lib/tasks/backup_restore.rake]: backup_dir = "/home/beianca/db_backups" change it to  backup_dir = "/tmp/db_backups"

**Pull request merged**